Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/main/home/HomeFragmentViewModel.kt

```diff
@@ -83,12 +85,17 @@ class HomeFragmentViewModel @AssistedInject constructor(
     private val deadmanNotificationScheduler: DeadmanNotificationScheduler,
     private val appShortcutsHelper: AppShortcutsHelper,
     private val tracingSettings: TracingSettings,
+    private val traceLocationOrganizerSettings: TraceLocationOrganizerSettings
 ) : CWAViewModel(dispatcherProvider = dispatcherProvider) {
 
     private val tracingStateProvider by lazy { tracingStateProviderFactory.create(isDetailsMode = false) }
 
+    private val mutableIsQrInfoAcknowledged = MutableLiveData<Boolean>()
+    val isQRInfoAcknowledged: LiveData<Boolean> = mutableIsQrInfoAcknowledged
```

Comment:
`mutableIsQrInfoAcknowledged` and `isQRInfoAcknowledged` are redundant. Values are not used as livedata. You can return the value from `wasQRInfoWasAcknowledged()` directly.

Reply:
Yes, you are right. I'll have it changed.